lib/serverengine/process_manager.rb

```diff
@@ -146,9 +146,12 @@ def spawn(*args)
       rpipe, wpipe = new_pipe_pair
 
       begin
-        options[[wpipe.fileno]] = wpipe
-        if @enable_heartbeat
-          env['SERVERENGINE_HEARTBEAT_PIPE'] = wpipe.fileno.to_s
+
+        unless $platformwin
```

Comment:
I add heartbeat pipe for windows.
https://github.com/naritta/serverengine/commit/78a87445379a22274d1736104c423137343c00c9

In windows, it is impossible to share pipe easily like unix. 
Then I used namedpipe for windows and share the name of the namedpipe with env like unix.